packages/fxa-auth-client/lib/client.ts

```diff
@@ -272,42 +297,15 @@ export default class AuthClient {
   async signIn(
```

Comment:
It probably won't be terribly obvious to most why we have the different signin methods. It would be nice to have some auto documentation here, but not in scope.